wicd normally makes the NETWORKS directory at install time, but we cannot do that,
so we make it at runtime instead.

--- wicd-1.6.2.2/in/scripts=wicd.in	2009-09-01 11:05:31.000000000 -0400
+++ wicd-1.6.2.2/in/scripts=wicd.in	2010-01-04 18:01:57.000000000 -0500
@@ -1,3 +1,5 @@
 #!/bin/bash
 
+mkdir -p %NETWORKS%
+
 exec python -O %LIB%wicd-daemon.py $@
